asset/quickstart/analyze-iam-policy-longrunning-bigquery/main.go

```diff
@@ -0,0 +1,72 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START asset_quickstart_analyze_iam_policy_longrunning_bigquery]
+
+// Sample analyze_iam_policy_longrunning analyzes accessible IAM policies that match a request.
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"log"
+
+	asset "cloud.google.com/go/asset/apiv1"
+	assetpb "google.golang.org/genproto/googleapis/cloud/asset/v1"
+)
+
+func main() {
+	scope := flag.String("scope", "", "Scope of the analysis.")
+	fullResourceName := flag.String("fullResourceName", "", "Query resource.")
+	dataset := flag.String("dataset", "", "Output Bigquery Dataset.")
+	tablePrefix := flag.String("tablePrefix", "", "Output Bigquery table prefix.")
+	flag.Parse()
+	ctx := context.Background()
+	client, err := asset.NewClient(ctx)
+	if err != nil {
+		log.Fatalf("asset.NewClient: %v", err)
+	}
+	defer client.Close()
+
+	req := &assetpb.AnalyzeIamPolicyLongrunningRequest{
+		AnalysisQuery: &assetpb.IamPolicyAnalysisQuery{
+			Scope: *scope,
+			ResourceSelector: &assetpb.IamPolicyAnalysisQuery_ResourceSelector{
+				FullResourceName: *fullResourceName,
+			},
+			Options: &assetpb.IamPolicyAnalysisQuery_Options{
+				ExpandGroups:     true,
+				OutputGroupEdges: true,
+			},
+		},
+		OutputConfig: &assetpb.IamPolicyAnalysisOutputConfig{
+			Destination: &assetpb.IamPolicyAnalysisOutputConfig_BigqueryDestination{
+				BigqueryDestination: &assetpb.IamPolicyAnalysisOutputConfig_BigQueryDestination{
+					Dataset:     *dataset,
+					TablePrefix: *tablePrefix,
+				},
+			},
+		},
+	}
+
+	op, err := client.AnalyzeIamPolicyLongrunning(ctx, req)
+	if err != nil {
+		log.Fatalf("AnalyzeIamPolicyLongrunning: %v", err)
+	}
+	fmt.Print(op.Metadata())
```

Comment:
Should this wait for the operation to finish? Same with other samples.

Reply:
I intended to not wait for that. Here we printed the longrunning operation metadata, and in the test we only validate the metadata.